pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -246,18 +257,22 @@ func (e *BootstrapSigner) getConfigMap() *v1.ConfigMap {
 		return nil
 	}
 
-	if exists {
-		return configMap.(*v1.ConfigMap)
-	}
-	return nil
+	return configMap
+
 }
 
 func (e *BootstrapSigner) listSecrets() []*v1.Secret {
-	secrets := e.secrets.List()
+	secrets, err := e.secretLister.List(labels.Everything())
```

Comment:
I think you should be listing only in the appropriate namespace, right? This lists across all namespaces.